packages/http-client-python/eng/scripts/ci/util.py

```diff
@@ -69,4 +69,4 @@ def run_check(name, call_back, log_info):
         response = call_back(dirs[0])
     if not response:
         logging.error("%s fails", log_info)
-        exit(0)
+        exit(1)
```

Comment:
When call_back fails, the script shall exit with non-zero code to make sure CI check will fail.